Sources/SnapshotTesting/AssertSnapshot.swift

```diff
@@ -30,17 +30,49 @@ public func assertSnapshot<Value, Format>(
   testName: String = #function,
   line: UInt = #line
   ) {
+  assertSnapshot(matching: value,
+                 as: [snapshotting],
+                 named: name,
+                 record: recording,
+                 timeout: timeout)
```

Comment:
Thanks for the PR! Stephen and I will discuss this sometime soon, but in the meantime there is a quick thing you can do to get tests passing. Make sure to pass along the `file`, `testName` and `line` to the assert snapshot helpers so that files names are recorded properly.

We also prefer to have all the function arguments on a newline if they don't fit on online, as opposed to this hanging argument style.

Also there's an extra image that was added to the repo that could be deleted:

`assertSnapshot-matching-as-named-record-timeout-file-testName-line.1.png`

Reply:
Thanks for the comments, I will do the modification